src/snowflake/snowpark/modin/plugin/extensions/index.py

```diff
@@ -1329,36 +1366,24 @@ def is_integer(self) -> None:
         is_object : Check if the Index is of the object dtype. (deprecated).
         is_categorical : Check if the Index holds categorical data (deprecated).
         is_interval : Check if the Index holds Interval objects (deprecated).
-        """
-        # TODO: SNOW-1458123 implement is_integer
-
-    @index_not_implemented()
-    def is_interval(self) -> None:
```

Comment:
We should keep the method stub and raise NotImplementedError in case a user tries to call it.

Reply:
added